lib/wallaby/query.ex

```diff
@@ -289,4 +299,8 @@ defmodule Wallaby.Query do
         |> Keyword.put_new(:maximum, opts[:maximum])
     end
   end
+
+  defp add_get(opts) do
+    Keyword.put_new(opts, :get, nil)
```

Comment:
Looking at this is making me like `item` again. `at: :all` makes some kind of odd logic, but `item: :all` just reads so much better.